README.asciidoc

```diff
@@ -14,6 +14,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ////
+image::https://codecov.io/gh/Bit-Quill/tinkerpop/branch/master/graph/badge.svg?token=emILhOc6Rm[link="https://codecov.io/gh/Bit-Quill/tinkerpop"]
```

Comment:
I'd place this under the `== TinkerPop3` Title. See example https://github.com/apache/spark

Reply:
updated in ca00465